faq/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FAQ
+---
```

Comment:
Can we go ahead and start adding permalinks to the front-matter? If so please fix throughout.

`permalink: /faq/`